ksp_plugin/flight_plan.cpp

```diff
@@ -410,6 +437,12 @@ Status FlightPlan::ComputeSegments(
         anomalous_segments_ = 1;
         anomalous_status_ = status;
       }
+      coast_analysers_[it - manœuvres_.begin()]->RequestAnalysis(
```

Comment:
It's odd that we are doing this even if the coast is anomalous.  Same comment below.  If a coast is anomalous its analyser should stop (and those of the following coasts).

Reply:
No: if the flight plan’s synchronous computation of the coast times out before the end, it still makes sense to run the asynchronous analysis.

Analyses are not requested for coasts that are entirely past the anomaly though.